StripeConnect/StripeConnect/Source/Internal/Webview/MessageHandlers/AccountSessionClaimedMessageHandler.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  AccountSessionClaimedMessageHandler.swift
+//  StripeConnect
+//
+//  Created by Chris Mays on 8/14/24.
+//
+
+import Foundation
+
+// Emitted when the account session has been created.
+class AccountSessionClaimedMessageHandler: ScriptMessageHandler<AccountSessionClaimedMessageHandler.Message> {
+    struct Message: Codable, Equatable {
```

Comment:
nit: What do you think about calling this type `Payload` instead of `Message`?
 
I tend to think of "message" as being the entire message (including the name + payload) where as "payload" is the params sent in the message.